src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/filtering/MTLowHeteroplasmyFilterToolTest.java

```diff
@@ -0,0 +1,35 @@
+package org.broadinstitute.hellbender.tools.walkers.mutect.filtering;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.testutils.IntegrationTestSpec;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+
+public class MTLowHeteroplasmyFilterToolTest extends CommandLineProgramTest {
+    private static final File MITO_REF = new File(toolsTestDir, "mutect/mito/Homo_sapiens_assembly38.mt_only.fasta");
+    private static final File NA12878_MITO_FILTERED_VCF = new File(toolsTestDir, "mutect/mito/filtered.vcf");
+
+    @Test
+    public void testLowHetVariantWalker() throws IOException {
+        final IntegrationTestSpec testSpec = new IntegrationTestSpec(
+                        " -R " + MITO_REF.getAbsolutePath()  +
+                        " -V " + NA12878_MITO_FILTERED_VCF +
+                        " -O %s",
+                Arrays.asList(toolsTestDir + "mutect/mito/expected_LowHetVariantWalkerIntegrationTest_output.vcf")
+        );
+        testSpec.executeTest("testLowHetVariantWalker", this);
```

Comment:
I would really much rather have these tests read in the VCs and then check the filters (and check the relevant FILTER line in the header) rather than doing an exact match, because annoying things happen like someone changes the VCF header description of something unrelated and now these tests fail.  If that's going to be a huge pain, then we can live with this.  They're very thorough, I just want to save someone a little potential pain the future.